styles/style.css

```diff
@@ -0,0 +1,88 @@
+body{
+  background-color: #FFFFFF;
+}
+header.site-header{
+  background-color: #444444;
+  display: block;
+  border-style: solid;
+}
+h1.name{
+  float: left;
+  text-decoration: none;
+}
+nav.main-nav{
+display: block;
+padding-left: 60%;
+}
+ul.nav-list{
+  padding-top: 5%;
+}
+a{
+text-decoration: none;
+color: black;
+}
+li{
+  display: inline-block;
+  padding-top: 5px;
+  padding-bottom: 15px;
+  padding-left: 30px;
+  padding-right: 30px;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    }
+#profile-picture{
+  width: 100%;
+  margin-top: 3%;
+}
+h1{
+  text-decoration: underline;
+}
+.subbanners{
+  padding-top: 15px;
+  padding-bottom: 15px;
+  max-width: 1024px;
+  margin: auto;
+}
+.imageone{
+margin-right: 10px;
+}
+.imagetwo{
+margin-left: 15px;
+margin-right: 15px;
+}
+.imagethree{
+margin-left: 10px;
+  }
+section{
+  max-width: 1024px;
+  margin: auto;
+}
+.articleimage{
+float: left;
+}
+article{
+width: 450px;
+height: 320px;
+display: inline-block;
+padding-left: 20px;
+overflow: auto;
+}
+h3{
+font-size: 25px;
+}
+p{
+font-size: 20px;
+font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+button {
+display: block;
+margin: auto;
+background: black;
+color: white;
+}
+footer{
+  background-color: #444444;
+  display: block;
+  border-style: solid;
+  margin: auto;
+  height: 100px;
+}
```

Comment:
Nice styles! Watch our for indentation and line spacing inconsistencies. Also, try to add one media query and think mobile-first when updating these styles.